tutorials/dataframe/df103_NanoAODHiggsAnalysis.C

```diff
@@ -284,16 +283,16 @@ RNode reco_higgs_to_4el(RNode df)
 }
 
 // Compute mass of two Z candidates from two electrons and two muons and sort ascending in distance to Z mass
-RVec<float> compute_z_masses_2el2mu(rvec_f el_pt, rvec_f el_eta, rvec_f el_phi, rvec_f el_mass, rvec_f mu_pt,
-                                  rvec_f mu_eta, rvec_f mu_phi, rvec_f mu_mass)
+ROOT::RVecF compute_z_masses_2el2mu(cRVecF el_pt, cRVecF el_eta, cRVecF el_phi, cRVecF el_mass, cRVecF mu_pt,
+                                  cRVecF mu_eta, cRVecF mu_phi, cRVecF mu_mass)
```

Comment:
here

Reply:
so isn't that following previous standard?

Reply:
the second line should be indented a couple of columns more to the right. passing clang-format on your commit (not on the whole file) should fix it.